lib/elixir/lib/tuple.ex

```diff
@@ -1,60 +1,24 @@
 defmodule Tuple do
   @moduledoc """
   Functions for working with tuples.
-
-  Tuples are composite data types with a fixed number of elements. Tuples
-  can contain elements of any type, and a tuple can contain elements of
-  different types. Curly braces can be used to create tuples:
-
-      iex> {}
-      {}
-      iex> {1, :two, "three"}
-      {1, :two, "three"}
-
-  Tuples store elements contiguously in memory. This means accessing a
-  tuple element by index doesn't depend on the number of elements in the
-  tuple. We say the operation is done in constant-time, via the
-  `Kernel.elem/2` function:
-
-      iex> tuple = {1, :two, "three"}
-      iex> elem(tuple, 0)
-      1
-      iex> elem(tuple, 2)
-      "three"
-
-  Same goes for getting the tuple size with `Kernel.tuple_size/1`:
-
-      iex> tuple_size({})
-      0
-      iex> tuple_size({1, 2, 3})
-      3
-
-  Tuples being stored contiguously in memory also means that updating a tuple
-  (for example replacing an element with `Kernel.put_elem/3`) will make a
-  shallow copy of the whole tuple. The tuple elements are still shared thanks
-  to immutability.
-
-  Tuples are not meant to be used as a "collection" type but rather as a
-  fixed-size container for multiple elements. That's why it is not possible
+  
+  Please note the following functions for tuples--_all constant-time_
+  --are found in `Kernel`:
```

Comment:
I don't think this is the place to mention that these operations are constant-time. Let the documentation for each of those function mention that.